rengobot.py

```diff
@@ -546,14 +554,11 @@ async def background_task():
         except ConnectionResetError:
             print("Connection error")
 
-bot.loop.create_task(background_task())
```

Comment:
Also had to make these changes to run it locally.  Feel free to revert.  This is the recommended way to do this on the Discord dev site, so I'm hoping it works for you too.